plugins/search-backend-node/package.json

```diff
@@ -10,6 +10,9 @@
     "main": "dist/index.cjs.js",
     "types": "dist/index.d.ts"
   },
+  "backstage": {
+    "role": "plugin-backend"
```

Comment:
Perhaps `node-library`?  Basically only responsible for wiring up search engine / indexing; doesn't expose routes.

Reply:
Yep, makes sense 👍